controllers/SitemapController.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+/**
+ * Pimcoresitemapplugin_SitemapController
+ *
+ * @author Yann Weyer
+ */
+
+use Byng\Pimcore\Sitemap\SitemapPlugin as SitemapPlugin;
+
+class Pimcoresitemapplugin_SitemapController extends \Pimcore\Controller\Action\Frontend
```

Comment:
Can the `Frontend` class be `use`d at the top of the file, and can this class be made final please?